rpcserver.go

```diff
@@ -916,6 +916,23 @@ func (r *rpcServer) QueryAddrs(ctx context.Context,
 	}, nil
 }
 
+// parseAssetVersion parses an asset version from the RPC variant.
+func parseAssetVersion(version taprpc.AssetVersion) (asset.Version, error) {
```

Comment:
In many other places in our code we call this "unmarshal". We should consider doing the same here.

I think this function should be in this file: `taprpc/marshal.go`

Reply:
Yep this has been moved, as needed to add some other funcs, and now in the `taprpc` pacakge.